ndarray/src/main/java/org/tensorflow/ndarray/SparseNdArray.java

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.ndarray;
+
+/**
+ * Interface for Sparse Arrays
+ *
+ * @param <T> the type that the array contains
+ * @param <U> the type of dense NdArray
+ */
+public interface SparseNdArray<T, U extends NdArray<T>> extends NdArray<T> {
+  /**
+   * Gets the Indices
```

Comment:
Maybe a more detailed description of what are `indices` and `values` of a sparse array here?

I guess you can move the doc added to the private fields of `AbstractSparseNdArray` to this interface instead.

Reply:
OK, I copied the descriptions  from `AbstractSparseNdArray`.